src/functionalTest/java/uk/gov/hmcts/reform/pip/data/management/controllers/lists/PublicationIacDailyListTest.java

```diff
@@ -104,8 +106,9 @@ void testCreationOfValidIacDailyList() throws Exception {
     }
 
     @DisplayName("Should return an error message back to the user when creating an invalid blob")
-    @Test
-    void testCreationOfInvalidIacDailyList() throws Exception {
+    @ParameterizedTest
+    @EnumSource(value = ListType.class, names = {"IAC_DAILY_LIST", "IAC_DAILY_LIST_ADDITIONAL_LIST"})
+    void testCreationOfInvalidSjpPublicList(ListType listType) throws Exception {
```

Comment:
The method name should be IAC list not SJP Public

Reply:
Updated method name